+++ vulkan_core.h	2026-03-02 14:38:47.422484009 +0100
@@ -8190,7 +8190,7 @@
     const VkVideoDecodeInfoKHR*                 pDecodeInfo);
 #endif
 
-
+/*
 #define VK_KHR_video_decode_h264 1
 #include "vk_video/vulkan_video_codec_h264std.h"
 #include "vk_video/vulkan_video_codec_h264std_decode.h"
@@ -8248,7 +8248,7 @@
     const void*                               pNext;
     const StdVideoDecodeH264ReferenceInfo*    pStdReferenceInfo;
 } VkVideoDecodeH264DpbSlotInfoKHR;
-
+*/
 
 
 #define VK_KHR_dynamic_rendering 1
@@ -9363,7 +9363,7 @@
 } VkPhysicalDeviceShaderClockFeaturesKHR;
 
 
-
+/*
 #define VK_KHR_video_decode_h265 1
 #include "vk_video/vulkan_video_codec_h265std.h"
 #include "vk_video/vulkan_video_codec_h265std_decode.h"
@@ -9414,7 +9414,7 @@
     const void*                               pNext;
     const StdVideoDecodeH265ReferenceInfo*    pStdReferenceInfo;
 } VkVideoDecodeH265DpbSlotInfoKHR;
-
+*/
 
 
 #define VK_KHR_global_priority 1
