+++ vulkan_beta.h	2026-03-02 14:41:21.440886821 +0100
@@ -49,7 +49,7 @@
 } VkPhysicalDevicePortabilitySubsetPropertiesKHR;
 
 
-
+/*
 #define VK_KHR_video_encode_queue 1
 #define VK_KHR_VIDEO_ENCODE_QUEUE_SPEC_VERSION 7
 #define VK_KHR_VIDEO_ENCODE_QUEUE_EXTENSION_NAME "VK_KHR_video_encode_queue"
@@ -544,7 +544,7 @@
     VkBool32                         useMaxFrameSize;
     VkVideoEncodeH265FrameSizeEXT    maxFrameSize;
 } VkVideoEncodeH265RateControlLayerInfoEXT;
-
+*/
 
 #ifdef __cplusplus
 }
